src/components/filter/filter.e2e.ts

```diff
@@ -226,4 +226,46 @@ describe("calcite-filter", () => {
       assertMatchingItems(await filter.getProperty("filteredItems"), ["regex"]);
     });
   });
+
+  describe("filter behavior with predefined value prop", () => {
+    let page: E2EPage;
+
+    beforeEach(async () => {
+      page = await newE2EPage();
+      await page.setContent(`<calcite-filter value="harry"></calcite-filter>`);
+      await page.evaluate(() => {
+        const filter = document.querySelector("calcite-filter");
+        filter.items = [
+          {
+            name: "Harry",
+            description: "developer",
+            value: "harry",
+            metadata: { haircolor: "red", favoriteBand: "MetallicA" }
+          },
+          {
+            name: "Matt",
+            description: "developer",
+            value: "matt",
+            metadata: { haircolor: "black", favoriteBand: "Radiohead" }
+          }
+        ];
+      });
+    });
+
+    function assertMatchingItems(filtered: any[], values: string[]): void {
```

Comment:
Can you move this helper one level up so that it can be shared between both `describe` groups instead of having 2 copies?